src/plugins/ConditionalRule.js

```diff
@@ -18,6 +19,7 @@ export default class ConditionalRule {
 
   constructor(selector: string, styles: Object, options: RuleOptions) {
     this.selector = selector
+    this.selectorId = selector.replace(' ', '')
```

Comment:
CSSKeyframesRule should have a name https://developer.mozilla.org/de/docs/Web/API/CSSKeyframesRule